.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*.c]
```

Comment:
.m

The codebase is fairly inconsistent using tabs for alignment, thus their size tends to matter.